### PR TITLE
Depricate persistence 

### LINKED ISSUE #2888: Remove "PERSISTENCE" Redis functionality and documentation
Currently the PERSISTENCE feature that integrates with Redis is in alpha state as per #2138, however it's currently not tested in the latest versions, and hence it would be best if the references from docs / code are removed in favour of re-thinking what is the best way in which we'd want to achieve persistence, and moreover whether we can just add an example that shows how this would work instead to gather further feedback towards how this can become a feature.

### PR DESCRIPTION
**Pull request by nadinet**
**What this PR does / why we need it**:
Remove "PERSISTENCE" Redis functionality and documentation.
"soft" deprication, flag is ignored.
**Which issue(s) this PR fixes**:

Fixes #2888

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

**Comment by seldondev** [comment]
[APPROVALNOTIFIER] This PR is **NOT APPROVED**

This pull-request has been approved by:
To complete the [pull request process](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process), please assign **axsaucedo**
You can assign the PR to them by writing `/assign @axsaucedo` in a comment when ready.

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=SeldonIO%2Fseldon-core).

<details open>
Needs approval from an approver in each of these files:

- **[OWNERS](https://github.com/SeldonIO/seldon-core/blob/master/OWNERS)**

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by adriangonz** [comment]
Nice one @nadinet! We may also need to re-generate the licenses information to remove all the `redis` licensing info.

**Comment by adriangonz** [comment]
@nadinet should we also remove the `--persistence` reference from the `s2i/bin/run` script in the S2I Python wrapper template?

https://github.com/SeldonIO/seldon-core/blob/228b41765fc57fda26f0b5d00526ff423d06206d/wrappers/s2i/python/s2i/bin/run#L47

I just realised that the same script also forces for the `PERSISTENCE` flag to be present, so maybe that's one we can also remove:

https://github.com/SeldonIO/seldon-core/blob/228b41765fc57fda26f0b5d00526ff423d06206d/wrappers/s2i/python/s2i/bin/run#L24-L30

**Comment by seldondev** [comment]
@nadinet: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-cnztr) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following test **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-cnztr) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-cnztr) | `/test integration`
notebooks | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-9j7kq) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-cnztr) | `/test integration`
notebooks | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-9j7kq) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-cnztr) | `/test integration`
notebooks | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-9j7kq) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-9j7kq) | `/test notebooks`
integration | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-8s7wd) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 749598c05ea95f80d9c434d5cd494db56c1c0257 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-9j7kq) | `/test notebooks`
integration | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-8s7wd) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-8s7wd) | `/test integration`
notebooks | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-t4f65) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

File: wrappers/s2i/python/s2i/bin/run
Comment (axsaucedo) [inline wrappers/s2i/python/s2i/bin/run:47]: Incorrect syntax

**Review by axsaucedo — CHANGES_REQUESTED** [review]

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-t4f65) | `/test notebooks`
integration | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-jf5cg) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-t4f65) | `/test notebooks`
integration | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-jf5cg) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f16e97e0d9ef0cf3eb03e448cd722cd4e7d1e6fc | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-t4f65) | `/test notebooks`
integration | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-jf5cg) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-jf5cg) | `/test integration`
notebooks | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-d49v7) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-jf5cg) | `/test integration`
notebooks | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-d49v7) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | f50b1a8e835842503c7117896d9f49b084d0a012 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-d49v7) | `/test notebooks`
integration | e2502b8d0c52a1eed42c30e7937203373e74be9b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-5hzdq) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | e2502b8d0c52a1eed42c30e7937203373e74be9b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-5hzdq) | `/test integration`
notebooks | e2502b8d0c52a1eed42c30e7937203373e74be9b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-lwjr6) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by axsaucedo** [comment]
It seems the tests have found a couple of issues:

On the integration tests it failed because mlflow server seems to re-install the older version of seldon core

image

image

Basically showing error importing redis

image

Fixed by removing extra installation, and uses the base dependency correctly

image


On the notebooks tests it's one test failed

image

The failed test also has an mlfow server run at the bottom, ran the rest of the examples successfully

image

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | e2502b8d0c52a1eed42c30e7937203373e74be9b | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-lwjr6) | `/test notebooks`
integration | d098d82484c938020efcd4d37519e202c41dfe4a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-mn89z) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | d098d82484c938020efcd4d37519e202c41dfe4a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-mn89z) | `/test integration`
notebooks | d098d82484c938020efcd4d37519e202c41dfe4a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-zkws6) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | d098d82484c938020efcd4d37519e202c41dfe4a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-zkws6) | `/test notebooks`
integration | d098d82484c938020efcd4d37519e202c41dfe4a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-zlfwr) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | d098d82484c938020efcd4d37519e202c41dfe4a | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-zkws6) | `/test notebooks`
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-sqj2d) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-sqj2d) | `/test integration`
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-r6t86) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-sqj2d) | `/test integration`
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-r6t86) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-r6t86) | `/test notebooks`
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-btxh4) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-btxh4) | `/test integration`
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-969jm) | `/test notebooks`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-969jm) | `/test notebooks`
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-p2m5h) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-969jm) | `/test notebooks`
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-cmjzk) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-969jm) | `/test notebooks`
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-qfngg) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by seldondev** [comment]
@nadinet: The following tests **failed**, say `/retest` to rerun them all:

Test name | Commit | Details | Rerun command
--- | --- | --- | ---
notebooks | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/notebooks-969jm) | `/test notebooks`
integration | 2e485ec483235af22cd8dff00ae03c84f52d21e3 | [link](http://dashboard-jx.34.66.17.27.nip.io/namespaces/jx/pipelineruns/integration-bv4lx) | `/test integration`



<details>

Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [jenkins-x/lighthouse](https://github.com/jenkins-x/lighthouse/issues/new?title=Command%20issue:) repository. I understand the commands that are listed [here](https://go.k8s.io/bot-commands).
</details>

**Comment by RafalSkolasinski** [comment]
Just to check if changes in original mlflow bucket managed to propagate
/test notebooks

**Pull request merged**
